lq1/default.cfg

```diff
@@ -111,4 +107,58 @@ alias +lq-zoom90 "fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fo
 alias -lq-zoom90 "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90"
 alias +lq-zoom120 "fov 110;wait;fov 100;wait;fov 90;wait;fov 80;wait;fov 70;wait;fov 60;wait;fov 50;wait;fov 40"
 alias -lq-zoom120 "fov 50;wait;fov 60;wait;fov 70;wait;fov 80;wait;fov 90;wait;fov 100;wait;fov 110;wait;fov 120"
+
+// Graphics Style Toggle
+alias lq_style_modern_exec "exec style_modern.cfg"
+alias lq_style_retro_exec "exec style_retro.cfg"
+alias lq_style_modern "lq_style_modern_exec; alias lq_toggle_style_set lq_style_retro"
+alias lq_style_retro "lq_style_modern_exec; alias lq_toggle_style_set lq_style_modern"
+alias lq_toggle_style "cl_warncmd 0; lq_toggle_style_set; exec autoexec.cfg; vid_reload; cl_warncmd 1"
+lq_style_modern
+
+// Other Config Files
+alias	"cam"		"exec cam.cfg"
+alias	"dev"		"exec dev.cfg"
+alias	"zoom90"	"exec zoom90.cfg"
+alias	"zoom120"	"exec zoom120.cfg"
+
+
+// ================================
+// Compatibility With Other Engines
+// ================================
+
+// Ironwail
+gl_texture_anisotropy 2
+gl_overbright_models 1
+vid_fsaa 2
+vid_fsaamode 1
+scr_pixelaspect 1
+scr_crosshairscale 3
+scr_menuscale 3
+cl_alwaysrun 0
+r_particles 2
+sv_autoload 0			# Disable automatic load on death
+
+// EZQuake

```

Comment:
Ditto, does ezquake have its own cfg it executes?

Reply:
See my above comment.